LeagueOfLegendApi/lol/src/App.js

```diff
@@ -0,0 +1,103 @@
+import { useState } from "react";
+import "./App.css";
+import axios from "axios";
+import Header from "./Components/Header";
+import Tier from "./Components/Tier";
+
+function App() {
+  const [summonerName, setSummonerName] = useState("");
+  const [summonerNameData, setSummonerNameData] = useState({});
+  const [summonerSoloLeagueData, setSummonerSoloLeagueData] = useState([]);
+  const [summonerFlexLeagueData, setSummonerFlexLeagueData] = useState([]);
+  const ApiKey = "RGAPI-2e0de1b6-bda0-4c5d-84f5-19173b23ed63";
+  function searchForSummoner(event) {
+    let NameApiCallString =
+      "https://kr.api.riotgames.com/lol/summoner/v4/summoners/by-name/" +
+      summonerName +
+      "?api_key=" +
+      ApiKey;
+
+    //name 보냄 -> id, puuid 받음
+    axios.get(NameApiCallString).then(function (nameData) {
+      setSummonerNameData(nameData.data);
+      const summonerId = nameData.data.id;
+      const summonerPuuid = nameData.data.puuid;
+      let LeagueApiCallString =
```

Comment:
```suggestion
      const LeagueApiCallString =
```

변경되지 않을게 예상되면 const 를 쓰는건 어떨까요?

Reply:
var, let, const 차이에 대해서 아시면 좋을 거 같아요.

호이스팅 개념도 알면 좋고요.

Reply:
gg